vrx_ign/models/wam-v/model.sdf.erb

```diff
@@ -70,15 +70,54 @@ end
   </plugin>
 
   <plugin
-    filename="libSurface.so"
-    name="vrx::Surface">
-    <link_name>base_link</link_name>
-    <vehicle_length>4.9</vehicle_length>
-    <vehicle_width>2.4</vehicle_width>
-    <hull_radius>0.213</hull_radius>
-    <fluid_level>0</fluid_level>
-
-    <!-- Waves -->
+    filename="libSimpleBuoyancy.so"
+    name="vrx::SimpleBuoyancy">
+    <fluid_density>1000</fluid_density>
+    <fluid_level>0.0</fluid_level>
+    <linear_drag>20.0</linear_drag>
```

Comment:
Looking at the code, I believe the linear and angular drag should be set to zero.

The SimpleHydrodynamics plugin for the WAMV applies drag forces at the robot/vessel level.   Linear and angular drag applied to the various buoyancy elements would be redundant and would overestimate the drag on the vessel.

Reply:
Agreed. I had this set to `0` in my teleop testing. [19d536c](https://github.com/osrf/vrx/pull/500/commits/19d536c20c950b825ac3c5bb0ee970eebddbfdce)